src/prefect/utilities/dockerutils.py

```diff
@@ -57,7 +57,7 @@ def get_prefect_image_name(
         flavor: An optional alternative image flavor to build, like 'conda'
     """
     parsed_version = Version(prefect_version or prefect.__version__)
-    is_prod_build = parsed_version.post is None
+    is_prod_build = parsed_version.local is None
```

Comment:
nice! can you tldr what's different about local?

Reply:
The `local` part is after the `+` (i.e. `3.0.10+4.gbacf23cbe2`). We'll get this back after reverting the version scheme change.